lib/money.js

```diff
@@ -57,9 +54,14 @@ export const renderMoney = renderAmount;
 /**
  * Render an amount with decimal separator but without currency symbol.
  * @param	 {object} money Money with amount property and optionally currency property.
+ * @param	{void|string} locale Locale to format the amount in.
  * @return {string} Formatted number.
  */
-export const renderNumberAmount = function (money) {
+export const renderNumberAmount = function (money, locale) {
+	const NUMBER_FORMATTER = new Intl.NumberFormat(locale, {
+		minimumFractionDigits: 2,
+		maximumFractionDigits: 2
+	});
 	return NUMBER_FORMATTER.format(money.amount);
 };
```

Comment:
You should probably save some CPU cycles and cache the NumberFormat the same way _getCurrencyFormatter does, for consistency's sake (and a micro performance boost).

Reply:
Added a cache like `_getCurrencyFormatter`.